server/bundler/utils.js

```diff
@@ -43,9 +43,22 @@ function getHashedUrl( filename ) {
 }
 
 function getCssUrls( css ) {
+	const leftToRightCSS = [];
+	const rightToLeftCSS = [];
+
+	if ( Array.isArray( css ) ) {
+		css.map( cssSection => {
+			leftToRightCSS.push( getHashedUrl( 'sections/' + cssSection + '.css' ) );
+			rightToLeftCSS.push( getHashedUrl( 'sections-rtl/' + cssSection + '.css' ) );
+		} );
+	} else {
+		leftToRightCSS.push( getHashedUrl( 'sections/' + css + '.css' ) );
+		rightToLeftCSS.push( getHashedUrl( 'sections-rtl/' + css + '.rtl.css' ) );
+	}
```

Comment:
We could take this problem from the opposite angle in order to simplify that code, i.e. we could simply make sure we always deal with an array:

```javascript
const cssSections = Array.isArray( css ) ? css : [ css ];

cssSections.map( cssSection => {
  leftToRightCSS.push( getHashedUrl( 'sections/' + cssSection + '.css' ) );
  rightToLeftCSS.push( getHashedUrl( 'sections-rtl/' + cssSection + '.rtl.css' ) );
} );
```

That's something we often do in the API code by the way.

Reply:
Thinking a bit more about it we could simplify that code further, and make it pure by introducing a new function:

```javascript
function getHashedUrls( cssSections, isRTL = false ) {
  return cssSections.map( ( cssSection ) => {
    const filename = isRTL ? `sections-rtl/${ cssSection }.rtl.css` : `sections/${ cssSection }.css`;

    return getHashedUrl( filename );
  } );
}

function getCssUrls( css ) {
  const cssSections = Array.isArray( css ) ? css : [ css ];

  return {
    ltr: getHashedUrls( cssSections ),
    rtl: getHashedUrls( cssSections, true )
  };
}
```

This is untested though :).

Reply:
This makes total sense and it works well. Updated to use the second version and introduced new function.